packages/formstr-app/src/containers/FormFillerNew/QuestionNode/InputFiller.tsx

```diff
@@ -40,24 +42,27 @@ export const InputFiller: React.FC<InputFillerProps> = ({
       [AnswerTypes.label]: <></>,
       [AnswerTypes.shortText]: (
         <Input
-          defaultValue={defaultValue as string}
+          value={defaultValue as string}
```

Comment:
what was the problem with default value? have we tested it doesn't break anything else? like in the formfiller experience?